include/c_types/ordering_rt.h

```diff
@@ -0,0 +1,55 @@
+/*PGR-GNU*****************************************************************
+File: ordering_rt.h
+
+Generated with Template by:
+Copyright (c) 2022 pgRouting developers
+Mail: project@pgrouting.org
+
+Function's developer:
+Copyright (c) 2022 Shobhit Chaurasia
+Mail: 000shobhitchaurasia at gmail.com
+
+------
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+********************************************************************PGR-GNU*/
+
+/*! @file */
+
+#ifndef INCLUDE_C_TYPES_ORDERING_RT_H_
+#define INCLUDE_C_TYPES_ORDERING_RT_H_
+#pragma once
+
+/* for int64_t */
+#ifdef __cplusplus
+#   include <cstdint>
+#else
+#   include <stdint.h>
+#endif
+
+/**************************************************************************
+ * return type for ordering
+ * ***********************************************************************/
+
+struct ORDERING_rt {
+    int s_vertex;
```

Comment:
Why `int`?
is s_vertex and identifier of a vertex? if so, then `int` is wrong and needs fixing.

otherwise give an explanation of why `int`

Same question for the `int` types on the other fields.